### PR TITLE
fix: adjusting contact redirects

### PR DESCRIPTION
**Pull request by githubdoramon**

### CODE REVIEW

**Review by JackHamer09 — APPROVED** [review]

File: firebase.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         },
         {
           "source": "/docs/contact.html",
-          "destination": "/docs/#footer",
```
Comment (Dendiiip) [inline firebase.json:53]: @Jinn

Comment (Dendiiip) [reply firebase.json:53]: > @Jinn

Hey

**Review by Dendiiip — COMMENTED** [review]
Bn  no l

File: firebase.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
...
+          "destination": "https://zksync.io/contact",
```
Comment (Dendiiip) [inline firebase.json:53]: Juju

**Review by Dendiiip — COMMENTED** [review]
Bn  no lol MMO

**Pull request merged**